lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -29,6 +29,33 @@
     "disableStorageReset": false,
     "emulatedFormFactor": "mobile",
     "channel": "cli",
+    "budgetsJSON": [
```

Comment:
It looks like you manually added this in? It's completely amazing you figured out this file even exists :) but it will run into the same issue as over in #8536, erased the next time someone updates the sample artifacts.

This file is generated by `yarn update:sample-artifacts`, which calls [`update-report-fixtures.js`](https://github.com/GoogleChrome/lighthouse/blob/552d7153cd06805b3457410cf6739bca7bc7b773/lighthouse-core/scripts/update-report-fixtures.js) to run lighthouse against [`dbw_tester.html`](https://github.com/GoogleChrome/lighthouse/blob/552d7153cd06805b3457410cf6739bca7bc7b773/lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html).

I *think* making this permanent should be as easy as
- adding this object to a budget file somewhere. Since the artifacts come from `dbw_tester.html`, maybe it makes sense to have the budgets file in `lighthouse-cli/test/fixtures/dobetterweb/budgets.json`?
- adding `--budgets-path=${pathToThatPlace}` to the `flags` in `update-report-fixtures.js`
- running `yarn update:sample-artifacts` to verify that the budget does indeed end up like this in this file
- `git checkout -- lighthouse-core/test/results/artifacts/*` all the generated files (devtoolslog.json, this file, etc) to get back to the state in this PR because the files will have a bunch of other changes we don't want right now :)

Reply:
we can also do this in a follow up if it's not actually easy to do this

Reply:
> we can also do this in a follow up if it's not actually easy to do this

I'd say +1 to this. I've manually updated/undone every programmatic change to artifacts.json to avoid unnecessary changes, so IMO this isn't a deal breaker

Reply:
SG

Reply:
Is this somehow related to why this branch is passing for me locally but failing remotely?

Reply:
@khempenius it's probably the master failures #8583  #7122 

I believe they're linux only, is your local Mac?

Reply:
Ah, makes sense. Yep, it's a Mac.